components/bsx/Offer/CollectionOffers.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <div>
+    <b-select v-model="selectedStatus">
+      <option
+        v-for="option in uniqType"
+        :value="option.type"
+        :key="option.type">
+        {{ option.value }}
+      </option>
+    </b-select>
+    <OfferTable
+      :offers="showOffers"
+      :accountId="accountId"
+      is-bsx-stats
```

Comment:
Why  using `is-bsx-stats` : )
how about controlling the display of specific column by props value like `displayCollection` `displayDate` ?

Reply:
Ok, I will use `displayCollection` to control collection column display.
However, I don't think that it's wise to control other columns by `displayXXX`. Because currently three components `OfferList.vue`, `OfferStats.vue` and `CollectionOffers.vue` are using `OfferTable.vue`, but the display column for each component is a lot of differences. If we control column by `displayXXX`, each component using `OfferTable.vue` needs to pass lots of Props to control, which will hard to maintain.

Reply:
I guess `is-bsx-stats` means it's in the stats page, but now it's used at other components likes `CollectionOffers `. It's a little bit hard to maintain now.